app/uk/gov/hmrc/exports/services/mapping/CachingMappingHelper.scala

```diff
@@ -34,6 +35,9 @@ class CachingMappingHelper {
       dangerousGoods = exportItem.dangerousGoodsCode.flatMap(_.dangerousGoodsCode).map(code => Seq(DangerousGoods(Some(code)))).getOrElse(Seq.empty)
     )
 
+
+  private def strip(description: String): String = description.replaceAll("(\\r\\n)|\\r|\\n", " " )
```

Comment:
I know I helped prototype this, but the method name could be better.   `stripCartiageReturn` perhaps.

Also, the method can be moved within the `commodityFromExportItem` method as that's the only place it's used.